pkgs/applications/version-management/gogs/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, makeWrapper
+, git, bash, openssh, pam
+, sqliteSupport ? true
+#, pamSupport ? false
+}:
+
+buildGoPackage rec {
+  name = "gogs-${version}";
+  version = "0.9.97";
+
+  src = fetchFromGitHub {
+    owner = "gogits";
+    repo = "gogs";
+    rev = "v${version}";
+    sha256 = "151mmd8h5zd4bvafd42nsky0m9gblf5fcpd6jacqcrry1796hxk9";
+  };
+
+  patchPhase = ''
+    substituteInPlace models/repo.go \
+      --replace '#!/usr/bin/env %s\n' "#!${bash}/bin/bash\\n#%s\\n"
+  ''; # replace shebang and ignore first argument (SCRIPT_TYPE in configuration) in TPL_UPDATE_HOOK
+
+  buildInputs = [ makeWrapper ]
+  #  ++ stdenv.lib.optional pamSupport pam
+  ;
+
+  buildFlags = "-tags"
```

Comment:
`buildFlags` should be a list if you specify multiple arguments, then you don't have to worry about the space prefix.